aten/src/ATen/native/native_functions.yaml

```diff
@@ -1966,6 +1974,14 @@
 
 - func: var(Tensor self, int[1] dim, bool unbiased=True, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
 
+- func: var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)
+  matches_jit_signature: True
```

Comment:
get rid of `matches_jit_signature`, it's the default now.

Reply:
Fixed